nginx-controller/controller/controller.go

```diff
@@ -626,7 +695,7 @@ func (lbc *LoadBalancerController) syncCfgm(task Task) {
 		ingExes = append(ingExes, ingEx)
 	}
 
-	if err := lbc.cnf.UpdateConfig(cfg, ingExes); err != nil {
+	if err := lbc.cnf.UpdateConfig(cfg, ingExes, mergeableIngresses); err != nil {
 		if cfgmExists {
```

Comment:
let's emit events for mergeableIngresses as well, including masters and minions

Reply:
I was thinking something, along these lines:

```
lbc.recorder.Eventf(ingEx.Ingress, api_v1.EventTypeWarning, "NotFound", "Master for %v/%v(Minion) was not found when processing configmap", ingEx.Ingress.Name, ingEx.Ingress.Namespace)
```

Reply:
@diazjf in the `syncCfgm`, it should be either `UpdatedWithError` or `Updated`, similarly to the regular Ingresses.  I think it is OK to skip other errors, like not founding the master.

Reply:
Done